index.html

```diff
@@ -9,7 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="container">
-  <img class="container-item book-cover" src="resources/book-cover.png" alt="book_cover">
+  <div class="container-item badge">
+    <img class="book-cover" src="resources/book-cover.png" alt="book_cover">
+  </div>
```

Comment:
maybe this does the same.
```suggestion
    <img class="book-cover container-item badge" src="resources/book-cover.png" alt="book_cover">
```

Reply:
the position relative passes up a level, namely, to the class container.